clinica/iotools/utils/pipeline_handling.py

```diff
@@ -196,7 +211,7 @@ def t1_freesurfer_longitudinal_pipeline(
                 )
                 atlas_df = pd.read_csv(atlas_path, sep="\t")
                 label_list = [
-                    "t1-freesurfer-segmentationVolumes_" + x
+                    "t1-fs-long_segmentationVolumes_" + x
```

Comment:
If a change of naming convention is necessary, it'd be nice to adopt a naming scheme which is BIDS compatible.

i.e. `[<prefix1>-<label1>]_[<prefix2>-<label2>]_[...]_<suffix>`

where `<prefix>` is purely alphabetic, `<label>` is alphanumerical and `<suffix>` alphanumerical. It looks to me that we are introducing entities without prefix, or with multiple `-` which are reserved for separating key-value pairs.